### ISSUE TITLE
OS version undefined - unable to debug

### ISSUE THREAD

**Issue by marekkalnik** [AppGyver/steroids#879]
Hi,

In my Steroids Window I see the OS version as undefined. The click on debug link does not work, because of an android error (cannot read property replace of undefined).

Host: Linux Ubuntu
Device: Nexus 4, Android 5.0
Steroids CLI version: v4.0.12
![os_undefined_android](https://cloud.githubusercontent.com/assets/548629/6771760/4f04d6d0-d0e9-11e4-9c56-3acb9bed289a.png)

**Comment by harsaharsa** [comment]
Have you enabled USB debugging for your device? You also need to connect it via an USB cable. Can you debug e.g. mobile websites running on your device via the `chrome://inspect` page?

**Comment by marekkalnik** [comment]
Yes, I can debug websites and the application using chrome://inspect it's just the debug button that does not work.

**Comment by harsaharsa** [comment]
Ah, right, I understand now. Labeled as bug.
